### ISSUE TITLE
Task - Collection Object Match - Feature Request 

### ISSUE THREAD

**Issue by AntWeb-org** [SpeciesFileGroup/taxonworks#3239]
### Feature or enhancement

Would like an 'undo' button option in case one or multiple CO match updates were incorrectly updated.

This along with the data attributes display (issue #3236) will help us find potential typos in CO catalog numbers or trip code identifiers, and keep us from updating the wrong CO.

### Location

Task - Collection Object Match

**Comment by AntWeb-org** [comment]
@bpescador, @cgrinter, @rich-keller

**Comment by mjy** [comment]
This is tricky as "undo" can mean many things and requires us to track a whole new state space.

I think, as you note, that as a first step we need to 1) improve ability in compare and 2) provide more information in a table-like form as the first line of defense.   We could also explore 3) providing some similiarity metrix to warn against possible outliers in combination with 4) requiring an extra confirmation for those outliners.

**Comment by AntWeb-org** [comment]
Seems like a good start. Looking forward to the next phase of this task.

**Comment by tmcelrath** [comment]
I agree that this would be good. I haven't run into the need for it yet but I'm sure I will eventually. I'm usually very careful about who can use this task and when they use it, which partially mitigates this issue. But I agree, an "undo" button would be useful in many places.

**Comment by AntWeb-org** [comment]
CO Match task no longer exists/replaced with Match collection objects/use filters instead to sort/update
